src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -1073,6 +1073,26 @@ def create_vm(cmd, vm_name, resource_group_name, image=None, size='Standard_DS1_
         return sdk_no_wait(no_wait, client.begin_create_or_update, resource_group_name, deployment_name, deployment)
     LongRunningOperation(cmd.cli_ctx)(client.begin_create_or_update(resource_group_name, deployment_name, deployment))
 
+    # Guest Attestation Extension and enable System Assigned MSI by default
+    if security_type and enable_vtpm and enable_secure_boot and not disable_integrity_monitoring:
```

Comment:
Please judge whether the value of `securityType` is `TrustedLaunch`

Reply:
`--security-type` has only one option.